modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/KeyValueStorage.java

```diff
@@ -237,15 +237,16 @@ public interface KeyValueStorage extends ManuallyCloseable {
      * Saves an arbitrary storage configuration as a byte array.
      *
      * @param configuration Configuration bytes.
-     * @param index Operation's index.
-     * @param term Operation's term.
+     * @param lastAppliedIndex Last applied index.
+     * @param lastAppliedTerm Last applied term.
      */
-    void saveConfiguration(byte[] configuration, long index, long term);
+    void saveConfigurationWithLastAppliedIndexAndTerm(byte[] configuration, long lastAppliedIndex, long lastAppliedTerm);
```

Comment:
It seems that it's enough to rename parameters. `saveConfiguration()` is ok as a name; the caller will not be confused about what kind of index+term they need to pass

Reply:
fixed